File: android-java/JavaMaps/app/src/main/java/com/tnqr/javamaps/model/Place.java
package com.tnqr.javamaps.model;

import androidx.room.ColumnInfo;
import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class Place {
    @PrimaryKey(autoGenerate = true)
    public int uid;

    @ColumnInfo(name="name")
    public String name;


    @ColumnInfo(name = "longitude")
    public double longitude;


    @ColumnInfo(name = "latitude")
    public double latitude;

    public Place(String name, double longitude, double latitude) {
        this.name = name;
        this.longitude = longitude;
        this.latitude = latitude;
    }
}

File: android-kotlin/GoogleMaps/app/src/main/java/com/example/googlemaps/database/Place.kt
package com.example.googlemaps.database

import android.graphics.Bitmap
import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey
import java.sql.Blob

@Entity
class Place (
    @PrimaryKey(autoGenerate = true)
    val id : Int = 0,
    @ColumnInfo(name = "place_name")
    val name : String,
    @ColumnInfo(name = "place_comment")
    val comment : String,
    @ColumnInfo(name = "place_image")
    val image : ByteArray,
    @ColumnInfo(name = "place_lng")
    val lng: Double,
    @ColumnInfo(name = "place_lat")
    val lat: Double
){

}